api_tests/institutions/views/test_institution_registrations_list.py

```diff
@@ -54,3 +55,31 @@ def test_return_private_nodes_mixed_auth(self):
         assert_in(self.registration1._id, ids)
         assert_in(self.registration2._id, ids)
         assert_in(self.registration3._id, ids)
+
+    def test_doesnt_return_retractions_without_auth(self):
+        self.registration1.retraction = RetractionFactory()
+        self.registration1.save()
+        self.registration1.retraction.state = Sanction.APPROVED
+        self.registration1.retraction.save()
```

Comment:
There is also a RetractedRegistrationFactory that can take care of a lot of this in one line. Arguments are the registration and a user that can approve the retraction.

If user3 is added as creator of registration1:

`retraction = RetractedRegistrationFactory(registration=self.registration1, user=self.user3)`.

Then you can get rid of 
```
self.registration1.retraction = RetractionFactory()
self.registration1.save()
self.registration1.retraction.state = Sanction.APPROVED
self.registration1.retraction.save()
```